torch/csrc/jit/passes/utils/memory_dag.cpp

```diff
@@ -1,12 +1,28 @@
 #include "memory_dag.h"
 
+#include <c10/util/flat_hash_map.h>
 #include <torch/csrc/utils/memory.h>
 #include <algorithm>
 #include <queue>
 
 namespace torch {
 namespace jit {
+ska::flat_hash_map<const Element*, int> comprMap;
+ska::flat_hash_map<int, const Element*> decomprMap;
 
+int getCompressed(const Element* x) {
```

Comment:
Is "compression" the right naming here? If it's a bijection from elements to indexes—maybe "toIndex" and "toElement" methods on `Element` is the right place for them?

Reply:
I agree with the naming - I'm thinking we can put them as static methods in `Element`? `convert` is a little bit clunky, and if we put them as static methods we can remove that.